### PR TITLE
Clean up Gallery block move/remove UI

### PR DESCRIPTION
**Pull request by kjellr**
This PR tidies up a couple small visual bugs with the Gallery block's individual image controls. It also switches the buttons to appear larger when possible. This is helpful (especially on mobile) to create a more visible hit area for these previously-tiny controls.

## 1. Button size improvements: 

**Before**

Screen Shot 2019-07-29 at 9 25 00 AM

**After**

Screen Shot 2019-07-29 at 9 24 18 AM

The buttons increase from 20 to 24px. Note that on larger screens, these buttons will shrink back down if there are more than 7 columns present. Is to prevent overlap: 

number-of-columns

## 2. Clean up extra `is-selected` border on the caption field.

I'm not sure when this made its first appearance, but I believe it's unintentional? 

**Before**

Screen Shot 2019-07-29 at 10 03 14 AM

**After**

Screen Shot 2019-07-29 at 9 48 00 AM

## 3. Removes box shadow from move/remove controls on hover and focus

These borders weren't working against the blue background.

**Before**

Screen Shot 2019-07-29 at 9 56 47 AM

**After**

Screen Shot 2019-07-29 at 9 56 26 AM

### CODE REVIEW

File: packages/block-library/src/gallery/editor.scss [hunk 1]
```diff
@@ -19,7 +19,7 @@ ul.wp-block-gallery {
 		outline: none;
 	}
 
-	.is-selected {
+	> .is-selected {
```
Comment (nosolosw) [inline packages/block-library/src/gallery/editor.scss]: This works and I'm fine with it. Wanted to run by you a different approach to see what you think: `figure.is-selected` (I'd also apply this to the `is-transient` rule below for clarification). Those seem to be the only two rules that don't have a block/element class along with the modifier class.

**Review by oandregal — APPROVED** [review]
Left one suggestion, in case you think it makes sense. Otherwise, this is good to go.

File: packages/block-library/src/gallery/editor.scss (hunk 1 above)
Comment (kjellr) [reply packages/block-library/src/gallery/editor.scss]: Sure, that makes sense. I'll push a quick update to do that.

**Comment by kjellr** [comment]
Thanks, everyone!

**Pull request merged**